internal/configs/configload/testdata/invalid-names-in-submodules/.terraform/modules/modules.json

```diff
@@ -0,0 +1,14 @@
+{
+	"Modules": [
+		{
+			"Key": "test",
+			"Source": "./sub",
+			"Dir": "testdata/invalid-names-in-submodules/sub"
```

Comment:
I was somewhat confused at first that all the modules in `testdata` have essentially fake `Dir` - if each dir was otherwise `terraform init`ed directly, there wouldn't be `testdata/invalid-names-in-submodules` but only the last part (`sub`).

I'm not sure if that's intentional or not, just something I wanted to point out here.